FogCreek/Glitch-Community
# Reminiscent airmail: hookify join team page

## Links
* Remix link: https://reminiscent-airmail.glitch.me/

## Changes:
* hookify join team page

## How To Test:
* log into https://reminiscent-airmail.glitch.me/
* visit https://reminiscent-airmail.glitch.me/join/@jungle-order/invalid-token
* you should be redirected to https://reminiscent-airmail.glitch.me/@jungle-order, and see an error notification
* have a user invite you to a team
* visit the link provided in the invitation email (using reminiscent-airmail.glitch.me instead of glitch.com)
* you should be invited to the team, and see an "invitation accepted" notification
